### PR TITLE
fix(docs): Add documentation for ldap authn edit command params.

### PR DESCRIPTION
**Pull request by avh121**

### CODE REVIEW

File: reference/halyard/commands.md [hunk 1]
```diff
@@ -4509,9 +4509,13 @@ hal config security authn ldap edit [parameters]
  * `--deployment`: If supplied, use this Halyard deployment. This will _not_ create a new deployment.
  * `--no-validate`: (*Default*: `false`) Skip validation.
  * `--url`: ldap:// or ldaps:// url of the LDAP server
- * `--user-dn-pattern`: Placeholder...uid={0},ou=users
- * `--user-search-base`: Placeholder...
- * `--user-search-filter`: Placeholder
+ * `--user-dn-pattern`: The pattern for finding a user's DN using simple pattern matching. For
```
Comment (lwander) [inline reference/halyard/commands.md:4512]: This page is auto-generated, I think you want to make these changes in halyard?

Comment (lwander) [reply reference/halyard/commands.md:4512]: The changes should be made here: https://github.com/spinnaker/halyard/blob/master/halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/security/authn/ldap/EditLdapCommand.java#L52-L64

Run `make` in the `halyard-cli` directory to regenerate the local docs. The next stable release of halyard will automatically update this page.

File: reference/halyard/commands.md [hunk 2]
```diff
@@ -4509,9 +4509,13 @@ hal config security authn ldap edit [parameters]
...
+   example, the pattern 'uid={0},ou=users' finds the user's DN by substituting the user's login name
+   in the supplied pattern.
+ * `--user-search-base`: The part of the directory tree under which user searches should be performed.
+ * `--user-search-filter`: The filter to use when searching for a user's DN. For example, the filter
```
Comment (dorbin) [inline reference/halyard/commands.md:4516]: I'm not 100% clear on the difference between this one and --user-dn-pattern. Is it that user-dn-pattern returns results and user-search-filter filters those results?

Comment (avh121) [reply reference/halyard/commands.md:4516]: Both actually return results, I believe. I tried to clarify it in the updated PR ([here](https://github.com/spinnaker/halyard/pull/886)) -- let me know if that makes more sense, or if I should clarify further.

**Comment by lwander** [comment]
Closing in favor of PR: https://github.com/spinnaker/halyard/pull/886

**Pull request closed without merging**